cpp/src/phonenumbers/phonenumbermatcher.cc

```diff
@@ -407,7 +406,9 @@ PhoneNumberMatcher::PhoneNumberMatcher(const PhoneNumberUtil& util,
       max_tries_(max_tries),
       state_(NOT_READY),
       last_match_(NULL),
-      search_index_(0) {
+      search_index_(0),
+      isInputInCleanUTF8_(true) {
```

Comment:
clean -> valid?

Reply:
Also var names in c++ should be underscores, not camel case, so is_input_in_clean_utf8_

Reply:
so perhaps: is_input_valid_utf8

Reply:
Done. +1